src/gadgets/mobile-styles/MediaWiki:Gadget-mobile-styles.css

```
@@ -411,6 +411,8 @@ html>body.skin-minerva .content {
 /* 修复列表对齐问题 */
 html>body.skin-minerva .content ol {
     list-style-position: outside;
+}
```

感谢bshd佬，不过这里记得要加一个空行哦
```suggestion
}

```